Content.Server/Notification/NotifyExtensions.cs

```diff
@@ -21,22 +21,32 @@ public static class NotifyExtensions
         /// <param name="range">
         ///     The range in which to search for players, defaulting to one screen.
         /// </param>
-        public static void PopupMessageOtherClients(this IEntity source, string message, IPlayerManager? playerManager = null, int range = 15)
+        /// <param name="except">
+        ///     An array of entities to not popup the message for.
+        /// </param>
+        public static void PopupMessageOtherClients(
+            this IEntity source,
+            string message,
+            IPlayerManager? playerManager = null,
+            int range = 15,
+            params IEntity[] except)
```

Comment:
This is kinda dirty and it should really use filters but I'm fine with it for now I guess.